src/algo/tensor_path.py

```diff
@@ -0,0 +1,191 @@
+from pygraphblas import Matrix
+from more_itertools import unique_everseen
+
+from src.grammar.rsa import RecursiveAutomaton
+from src.graph.label_graph import LabelGraph
+
+
+def get_elements(s: set, n):
```

Comment:
There is `itertools.islice`, which does almost the same. **Notice**, that Iterating over a set happens in unsorted order, so maybe you wanted to do something different.